src/components/presentational/ValueSelector/ValueSelector.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { CSSProperties, useContext, useEffect, useState } from 'react';
+import './ValueSelector.css'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircle } from '@fortawesome/free-regular-svg-icons';
+import { faCheckCircle } from '@fortawesome/free-solid-svg-icons';
+import { ColorDefinition, resolveColor } from '../../../helpers/colors';
+import { LayoutContext } from '../Layout';
+
+export interface ValueSelectorProps {
+	backgroundColor?: ColorDefinition;
```

Comment:
Quick thing that jumped out at me - I have mostly left component backgrounds up to the container, since Card allows you to define a background color.  Is there a reason this should have a separate background color?

Reply:
We can probably drop that.  I had put it there because the default colors (white buttons) requires a gray background to work.  I added the override property because I figured if I was setting a default color in CSS, there should be a way to override it.

Reply:
I removed the background color and override property.  It now relies on the containing element for its background color.